packages/winpanda/extra/src/winpanda/core/cmdconf.py

```diff
@@ -102,6 +102,11 @@ def __init__(self, **cmd_opts):
         self.dcos_conf = self.get_dcos_conf()
         LOG.debug(f'{self.msg_src}: dcos_conf: {self.dcos_conf}')
 
+        # Add discovery configuration for dcos_conf
+        discovery_type = self.dcos_conf.get('values').get(
+            'master_discovery', 'master_http_loadbalancer')
```

Comment:
You can use indexing in both cases here instead of `get`:

`master_discovery` is a required configuration variable, so you can assume it exists, and fail if it does not.

`values` must exist, otherwise `None.get()` will raise an exception.

Reply:
changed